src/_guides/language/sound-problems.md

```diff
@@ -29,15 +29,14 @@ make sure that you're using the latest version of Dart.
 Alternatively, try adding the following code to a file:
 
 {:.fails-sa}
-<?code-excerpt "strong/lib/common_problems_analysis.dart (is-strong-mode-enabled)"?>
 {% prettify dart tag=pre+code %}
 bool b = [0][0];
 {% endprettify %}
 
 With type-safe Dart, the analyzer produces the following error:
 
 {:.console-output}
-<?code-excerpt "strong/analyzer-results-stable.txt" retain="/'int' can't be .* 'bool'.*common_problems/"?>
+<!-- code-excerpt "strong/analyzer-results-stable.txt" retain="/'int' can't be .* 'bool'.*common_problems/" -->
```

Comment:
Can we uncomment the code-excerpt tag? Otherwise, the analyzer message could change and we wouldn't know.

Reply:
The problem is that since we removed the files that generate those errors, we can't get that error excerpt from those files (same applies to the other comments about this)

Reply:
Well drat. I think it's fine for now, but since analysis messages do change, I could file a bug about fixing that.

Or... since analyzer messages are now listed & explained in https://dart.dev/tools/diagnostic-messages, for some messages at least we could have static (not included) text that links to the message description there (in another PR). An advantage of this approach is that if/when the message changes, @bwilkerson has started adding forwarding links to the old links. (See the discussion at #2569.)